charts/bulk-scan-processor/values.aat.template.yaml

```diff
@@ -5,3 +5,4 @@ java:
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
   environment:
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    DELETE_COMPLETE_FILES_CRON: "0/10 * * * * *"
```

Comment:
Is a reason we need this here. This config will only apply to staging

Reply:
Removed

Reply:
we should not have any over rides here as this is only for staging

Reply:
It is "0 0/10 * * * *" in values.yaml